doc/nrf/releases_and_maturity/migration/nRF54H20_migration_2.7/migration_guide_2.4.99-cs3_to_2.7_application.rst

```diff
@@ -74,6 +74,38 @@ General
 
      The previously used SOC1-based board files have been removed.
 
+nRF Desktop
+-----------
+
+You need to make the following changes for :ref:`nrf_desktop` application to work in the same way as in the release v2.4.99-cs3.
+
+* Due to the migration to the new hardware model, commonly referred as "hardware model v2", the board target of the nRF54H20 DK has changed to ``nrf54h20dk/nrf54h20/cpuapp``.
+  The build command for the application has changed and it is now:
+
+  .. code-block:: console
+
+    west build -b nrf54h20dk/nrf54h20/cpuapp
+
+* The :ref:`ipc_radio` image serves purpose as universal network core image for hci_ipc rpc_host and IEEE 802.15.4 remote image.
+  Due to that radio core now uses the :ref:`ipc_radio` application from ``sdk-nrf`` instead of the :ref:`zephyr:bluetooth-hci-ipc-sample` sample from ``sdk-zephyr``.
```

Comment:
```suggestion
  Due to this, the radio core now uses the :ref:`ipc_radio` application from ``sdk-nrf`` instead of the :ref:`zephyr:bluetooth-hci-ipc-sample` sample from ``sdk-zephyr``.
```